setup.cfg

```diff
@@ -54,15 +54,15 @@ docs =
 [options.package_data]
 sunkit_pyvista = data/*
 
-[tool:pytest]
-testpaths = "sunkit_pyvista" "docs"
-norecursedirs = ".tox" "build" "docs[\/]_build" "docs[\/]generated" "*.egg-info" "examples" ".history"
-doctest_plus = enabled
-doctest_optionflags = NORMALIZE_WHITESPACE FLOAT_CMP ELLIPSIS
-text_file_format = rst
-addopts = --doctest-rst
-markers =
-    display_server: marks this test function as needing a display server.
+# [tool:pytest]
+# testpaths = "sunkit_pyvista" "docs"
+# norecursedirs = ".tox" "build" "docs[\/]_build" "docs[\/]generated" "*.egg-info" "examples" ".history"
+# doctest_plus = enabled
+# doctest_optionflags = NORMALIZE_WHITESPACE FLOAT_CMP ELLIPSIS
+# text_file_format = rst
+# addopts = --doctest-rst
+# markers =
+#     display_server: marks this test function as needing a display server.
```

Comment:
What was this for?

Reply:
My bad, this was to run pytest locally, this shouldn't be here.